modules/appeals_api/app/swagger/notice_of_disagreements/v0/swagger_dev.json

```diff
@@ -699,48 +739,6 @@
                                     ]
                                   }
                                 },
-                                "claimant": {
```

Comment:
This field shouldn't have been removed - it's present in all our v2 10182 files, so I'm not sure why it disappeared here...